src/utils/DockerUtil.js

```diff
@@ -20,20 +20,29 @@ export default {
       throw new Error('Falsy ip or name passed to docker client setup');
     }
 
-    let certDir = path.join(util.home(), '.docker/machine/machines/', name);
-    if (!fs.existsSync(certDir)) {
-      throw new Error('Certificate directory does not exist');
-    }
+    switch(process.platform) {
```

Comment:
Oh .. sorry @jeffdm I have just noticed your comment here ... Let me do the update.